docs/scos/dev/glue-api-guides/202311.0/create-glue-api-applications.md

```diff
@@ -14,10 +14,10 @@ New Glue projects can create API applications. This is what you need to do in or
 
 Because the backend and storefront APIs have different sets of configured services, they also have different settings in Docker and different containers. Ensure your `deploy.yml` file contains the correct setting for `application`. The available options are listed in the following table:
```

Comment:
Why is it now called backend and storefrontAPIs while in "What is a API convention" its called Glue Storefront API and Glue Backend API?